test/e2e/flowaggregator_test.go

```diff
@@ -62,20 +62,20 @@ DATA SET:
     reversePacketDeltaCount: 136211
     reverseOctetDeltaCount: 7083284
     sourcePodName: perftest-a
-    sourcePodNamespace: antrea-test
+    sourcePodNamespace: testflowaggregator-b6mjmbpl
```

Comment:
but the namespace is generated randomly now, does this work?

Reply:
this part is just an example output as a comment, not the test code.

Reply:
got it, thanks